source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
+
+* Fedora 22
+* Fedora 21
+* Fedora 20
```

Comment:
Fedora 20 is EOL, so we aren't going to build any new packages for it. But since these are puppet 4.3 docs, that might not be an applicable point